spec/models/survey_spec.rb

```diff
@@ -1,3 +1,31 @@
 require 'rails_helper'
 
-describe Survey do end
+describe Survey do
+  describe 'formatted_date' do
+    it 'returns the formatted date' do
+      date = DateTime.current
+      survey = create :survey, date: date
+      expect(survey.formatted_date).to eql date.strftime('%m/%d/%Y')
```

Comment:
Can you express this format in a way other than reusing the strftime format string?